.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,38 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master", branch/* ]
+  pull_request:
+    branches: [ "master", branch/* ]
+  schedule:
+    - cron: '26 9 * * 6'
```

Comment:
Is the cron schedule necessary if this is going to run on every push and PR?

Reply:
Probably not with our activity. A weekly job is provided by default.

https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#scanning-on-a-schedule

It's generally meant for projects that may not be committing all the time, just to make sure that CodeQL is running, as new queries and such pop up often.

I'll go ahead and remove it.